src/main/java/uk/gov/ons/census/action/schedule/CaseClassifier.java

```diff
@@ -45,21 +50,33 @@ public void enqueueCasesForActionRule(ActionRule actionRule) {
           "INSERT INTO actionv2.case_to_process (batch_id, batch_quantity, action_rule_id, "
               + "caze_case_ref) SELECT ?, COUNT(*) OVER (), ?, case_ref FROM "
               + "actionv2.cases "
-              + buildWhereClause(actionRule.getActionPlan().getId(), actionRule.getClassifiers()),
+              + buildWhereClause(
+                  actionRule.getActionPlan().getId(),
+                  actionRule.getClassifiers(),
+                  actionRule.getActionType().getHandler()),
           batchId,
           actionRule.getId());
     }
   }
 
-  private String buildWhereClause(UUID actionPlanId, Map<String, List<String>> classifiers) {
+  private String buildWhereClause(
+      UUID actionPlanId, Map<String, List<String>> classifiers, ActionHandler actionHandler) {
     StringBuilder whereClause = new StringBuilder();
     whereClause.append(String.format("WHERE action_plan_id='%s'", actionPlanId.toString()));
     whereClause.append(" AND receipt_received='f'");
-    whereClause.append(" AND refusal_received='f'");
     whereClause.append(" AND address_invalid='f'");
     whereClause.append(" AND case_type != 'HI'");
     whereClause.append(" AND skeleton='f'");
 
+    if (actionHandler == ActionHandler.PRINTER) {
+      whereClause.append(
+          " AND (refusal_received IS NULL OR refusal_received='"
+              + RefusalType.HARD_REFUSAL.toString()
```

Comment:
it would be more efficient to write this as `refusal_received != RefusalType.EXTRAORDINARY_REFUSAL`

Reply:
although, I do like that you're using the enum, so maybe keep that part